### PR TITLE
Add machine parsable info.json files

### PR DESCRIPTION
**Pull request by Ayrx**
This PR adds an `info.json` file to each EFDA sub-project containing the number of vulnerabilities, direct dependencies and transitive dependencies.

### CODE REVIEW

File: golang/glide/only-glide.yaml/info.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "vulnerabilities": 2,
```
Comment (hendychua) [inline golang/glide/only-glide.yaml/info.json]: According to srcclr scan output, there are 3 vulnerabilities:
```
Vulnerabilities - Public Data
CVE-2016-9121                             High Risk       Invalid Curve Attack                                    github.com/square/go-jose 7f0dd80...
CVE-2016-9122                             Medium Risk     Signature Exploitation                                  github.com/square/go-jose 7f0dd80...

Vulnerabilities - Premium Data
NO-CVE                                    Medium Risk     Denial Of Service (DoS) Through An Integer Overflow     github.com/square/go-jose 7f0dd80...
```
Is this a typo or did you mean to count only public data? I think you should also include premium data if they are correct. You should verify it manually (check the registry page then follow the description and source code to verify the vulnerability).

File: java/jars/fat-jars/info.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "vulnerabilities": 3,
```
Comment (hendychua) [inline java/jars/fat-jars/info.json:2]: This is the project where owasp detected 1 more CVE than SourceClear. SourceClear has updated their database and it now shows 4 vulnerabilities. This has to be updated along with the [README](https://github.com/srcclr/efda/blob/master/java/jars/fat-jars/README.md). As with all other data, you should verify them manually first.

**Review by hendychua — CHANGES_REQUESTED** [review]
Looks good overall but there are a couple of changes required. Also, I'd suggest verifying the vulnerabilities data manually (especially the premium ones), as part of the other project requirement.

File: golang/glide/only-glide.yaml/info.json (hunk 1 above)
Comment (Ayrx) [reply golang/glide/only-glide.yaml/info.json]: Ah sorry! This one is a typo, will fix it as soon so I can.

The data here is all from the numbers in the README.md of each EFDA project which I believe is accurate at the point in time you guys added it? It isn't from the available data of the SourceClear scanner.

**Review by hendychua — APPROVED** [review]

**Review by hendychua — APPROVED** [review]

**Pull request merged**